test/performance/broker-imc/continuous/200-broker-imc.yaml

```diff
@@ -101,6 +101,8 @@ spec:
             - "--expect-records=1"
             - "--mako-tags=channel=imc"
             - "--verbose"
+            - "--benchmark-key=5903682180743168"
+            - "--benchmark-name=Development - Broker Latency & Throughput"
```

Comment:
Does this need quotes around the value?

Reply:
Have tried it locally by printing out this benchmark name in the pod's log, seems it doesn't need quotes.

Reply:
Can we add a comment that describes where these values come from?

Reply:
added